src/BloomExe/CollectionTab/LibraryModel.cs

```diff
@@ -215,7 +219,29 @@ public void ExportDocFormat(string path)
 			{
 				File.Delete(path);
 			}
+#if __MonoCS__
+			// Linux (Trusty) LibreOffice requires slightly more metadata at the beginning
+			// of the file in order to recognize it as HTML.  Otherwise it opens the file
+			// as raw HTML (See https://silbloom.myjetbrains.com/youtrack/issue/BL-2276 if
+			// you don't believe me.)  I don't know any perfect way to add this information
+			// to the file, but sed seems to be universally available on Linux so that's
+			// what we'll use.
+			using (var p = new Process())
+			{
+				p.StartInfo = new ProcessStartInfo();
```

Comment:
If you're going to pull the whole file into C#'s address space anyway, could you just use File.ReadAllText() and then a C# replace? Seems you aren't using any RE 'magic'. It would be shorter and not depend on an external program. Also could be used on both platforms, assuming it is at least harmless to do the same thing on Windows.